pkg/proxy/winkernel/proxier.go

```diff
@@ -165,6 +165,10 @@ func newServiceInfo(svcPortName proxy.ServicePortName, port *api.ServicePort, se
 		onlyNodeLocalEndpoints = true
 	}
 
+	var stickyMaxAgeSeconds int
```

Comment:
you should set this to 100 (or some good value) here or else the timeout is going to default to zero.

Reply:
Yes, I know - If `service.Spec.SessionAffinity == api.ServiceAffinityClientIP`, `stickyMaxAgeSeconds` will be set to 10800, and if `service.Spec.SessionAffinity != api.ServiceAffinityClientIP`, `stickyMaxAgeSeconds` will be 0. 

Kube-proxy will ignore the value of `stickyMaxAgeSeconds` if `service.Spec.SessionAffinity != api.ServiceAffinityClientIP`. So, I don't see any problem? Any I missing something?

Reply:
I guess it doesn't really matter. It's a change in behavior from how it was previously though, I'd prefer to see a minimal change from the previous code's behavior.

Reply:
Sure, sounds fair. It's fixed now. PTAL.